style.css

```diff
@@ -472,6 +472,142 @@ hr {
   display: none;
 }
 
+/* Pop-Up */
+.model_content {
+  width: 95%;
+  height: calc(100vh - 20px);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #fff;
+  border-radius: 8px;
+  padding-bottom: 10px;
+  padding-top: 25px;
+  margin: 12px auto;
+}
```

Comment:
- [Optioanl] (`HIGHLY RECOMMENDED`). Good job addressing the changes previously requested, But I notice on a mobile simulation your modal overflows and there is no possibility to scroll up. This is a bad thing when it comes to UX. Please consider addressing this.

![Screenshot 2023-02-01 105523](https://user-images.githubusercontent.com/69077061/215984816-26004dec-4df6-47d7-96cc-ec822f20af21.png)